SitemapElementInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle;
+
+/**
+ * Models implementing this interface can indicate whether they should be listed in the sitemap or not.
+ *
+ * For PHPCR-ODM documents used with the Doctrine\Phpcr\Sitemap.. , make sure to call the property that stores
```

Comment:
"the Doctrine\Phpcr\Sitemap.." - can we give the full bundle-relative class name ehre?